### PR TITLE
List template layout in IE11

### PR DESCRIPTION
**Pull request by Cesarcs7**
This PR fixes the layout of the webchat lists elements which was broken in IE11 since it does not support grid layout, I have changed it to flex

Please test the Lists templates and make sure they are looking and working as before in the webchat channel in IE11, also test in other browsers.

### CODE REVIEW

File: src/plugins/messenger/MessengerPreview/components/MessengerListTemplate/components/MessengerListTemplateElement.tsx [hunk 1]
```diff
@@ -44,43 +43,45 @@ export const getMessengerListTemplateElement = ({ React, styled }: MessagePlugin
...
             }
         }
 
         return (
             <div role="listitem">
                 <Root
-                    role={default_action?.url ? "link" : undefined} 
+                    role={default_action?.url ? "link" : undefined}
                     onClick={default_action && (e => onAction(e, default_action))}
+                    style={default_action?.url ? { cursor: "pointer" } : {}}
```
Comment (sushmi21) [inline src/plugins/messenger/MessengerPreview/components/MessengerListTemplate/components/MessengerListTemplateElement.tsx:70]: Merge regression: This inline style can be removed. The same style is applied using class 'link'.

Comment (Cesarcs7) [reply src/plugins/messenger/MessengerPreview/components/MessengerListTemplate/components/MessengerListTemplateElement.tsx]: fixed

**Comment by sushmi21** [comment]
Previously, there was 8px gap between the columns of the grid (i.e. between the list item text and image). Now, we don't. If you have long list item titles, then it looks a little off. Please compare the two images.
Screenshot from 2021-04-22 17-49-26 Screenshot from 2021-04-22 17-55-21

**Comment by Cesarcs7** [comment]
> Previously, there was 8px gap between the columns of the grid (i.e. between the list item text and image). Now, we don't. If you have long list item titles, then it looks a little off. Please compare the two images.
> Screenshot from 2021-04-22 17-49-26 Screenshot from 2021-04-22 17-55-21

I fixed this the closer i could to make it look like the behaviour with the grid before.

**Review by sushmi21 — APPROVED** [review]
Looks good to me

**Review by pedily — COMMENTED** [review]
works fine :)

**Pull request merged**